src/components/chat/chat-interface.tsx

```diff
@@ -0,0 +1,105 @@
+"use client";
```

Comment:
Need to be deleted, this is only a next js thing and with astro we are using `client:load‘

Reply:
fixed